modules/calypso-redirects.php

```diff
@@ -0,0 +1,112 @@
+<?php
+
+if ( ! class_exists( 'Jetpack_Calypso_Redirects' ) ) :
+
+class Jetpack_Calypso_Redirects {
+
+	public function __construct() {
+		if ( ! get_option( 'jetpack_calypso_redirects_active' ) || ! is_admin() ) {
```

Comment:
How is this option going to get set?  It may make sense to add it as a "Jetpack" option, like was done for the "edit_links_calypso_redirect" option for "edit" links redirecting to calypso.  This was done so that it gets cleared automatically when/if the plugin was deleted.  

See https://github.com/Automattic/jetpack/pull/7732/files for reference